### ISSUE TITLE
catch the net::ERR_CONNECTION_TIMED_OUT 

### ISSUE THREAD

**Issue by cherepanovic** [use-hooks/react-hooks-axios#44]
Hello community, 

who to catch the error message so that I would not see it in dev console in a browser (?)
`
GET http://URL net::ERR_CONNECTION_TIMED_OUT`

thanks!

**Comment by gumitrathore** [comment]
yeah it is breaking, please fix this issue and catch the error when the server is offline.
